src/lib/components/event/event-details-link.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+
+  import Link from '$lib/holocene/link.svelte';
+  import type { CombinedAttributes } from '$lib/utilities/format-event-attributes';
+  import { type EventLinkType } from '$lib/utilities/get-single-attribute-for-event';
+  import {
+    routeForEventHistory,
+    routeForNexusEndpoint,
+    routeForTaskQueue,
+  } from '$lib/utilities/route-for';
+
+  export let value: string;
+  export let attributes: CombinedAttributes;
+  export let type: EventLinkType;
+  export let light = false;
+
+  $: ({ workflow, namespace } = $page.params);
+
+  $: executionLink = routeForEventHistory({
+    namespace,
+    workflow,
+    run: value,
+  });
+  $: taskQueueLink = routeForTaskQueue({ namespace, queue: value });
+  $: childWorkflowLink = routeForEventHistory({
+    namespace,
+    workflow: attributes.workflowExecutionWorkflowId,
+    run: attributes.workflowExecutionRunId,
+  });
+  $: nexusEndpointLink = routeForNexusEndpoint(value);
+
+  $: hrefs = {
+    execution: executionLink,
+    'task-queue': taskQueueLink,
+    'child-workflow': childWorkflowLink,
+    'nexus-endpoint': nexusEndpointLink,
+  } as Record<Exclude<EventLinkType, 'none'>, string>;
+
+  $: href = hrefs[type];
+</script>
+
+<Link class="truncate break-all" {href} {light}>
```

Comment:
Truncate doesn't appear to be working as expected and as a result the copy button doesn't show up ⬇️ 

![Screenshot 2024-10-02 at 11 58 55 AM](https://github.com/user-attachments/assets/c587ff2f-558a-447e-9cb2-fff49768dae0)

Reply:
I think I was too overzealous on the max-w properties for these details while attempting to make sure everything was visible without scrolling. I removed all of those, but everything should still be visible (unless some super long id or something) on xl. Let me know if you have some better thoughts here!

Reply:
I think this is fine for now, but there might be some clues for how we want to handle these in the `WorkflowDetail` component since we do something similar there ⬇️ 

![Screenshot 2024-10-02 at 3 29 51 PM](https://github.com/user-attachments/assets/7d441f4b-f1a3-4d71-86aa-822df316a5ad)